.eslintignore

```diff
@@ -5,3 +5,6 @@ node_modules/
 
 # Reported issue (must be excluded here since it fails on parsing): https://github.com/babel/babel-eslint/pull/777
 src/babel-preset-adeira/src/__tests__/__fixtures__/features/internal-slot-properties.js
+
+# flow-typed for shipit
```

Comment:
is there better way?

Reply:
Nothing I am aware of.

Reply:
I'd remove the comment though. It's not for Shipit. It's part of the project code.